src/main/java/seedu/address/logic/Logic.java

```diff
@@ -24,19 +24,19 @@ public interface Logic {
     CommandResult execute(String commandText) throws CommandException, ParseException;
 
     /**
-     * Returns the AddressBook.
+     * Returns the TaskWise.
      *
-     * @see seedu.address.model.Model#getAddressBook()
+     * @see seedu.address.model.Model#getTaskWise()
      */
-    ReadOnlyAddressBook getAddressBook();
+    ReadOnlyAddressBook getTaskWise();
 
-    /** Returns an unmodifiable view of the filtered list of persons */
-    ObservableList<Person> getFilteredPersonList();
```

Comment:
Should we refactor the `Person` class to `Task` instead? It would appear that a `Task` Model would be more appropriate here, seeing that it is a method that is invoked on the main model of the app.

I notice that this issue is present in other code files.